screengrab/spec/runner_spec.rb

```diff
@@ -36,7 +36,7 @@ def mock_adb_response(mock_response)
       end
       it 'sets custom launch_arguments' do
```

Comment:
Could we add more unit tests to test `use_timestamp_suffix`? 🙏 

* check if the default value is `true`
* set `use_timestamp_suffix` to `false`

🙇 🙇

Reply:
added 👍